programs/CMakeLists.txt

```diff
@@ -66,18 +66,18 @@ else()
     message(STATUS "Library bridge mode: OFF")
 endif()
 
-if (ENABLE_CLICKHOUSE_KEEPER)
-    message(STATUS "ClickHouse keeper mode: ON")
-else()
-    message(STATUS "ClickHouse keeper mode: OFF")
-endif()
-
 if (ENABLE_CLICKHOUSE_KEEPER_CONVERTER)
     message(STATUS "ClickHouse keeper-converter mode: ON")
 else()
     message(STATUS "ClickHouse keeper-converter mode: OFF")
 endif()
 
+if (ENABLE_CLICKHOUSE_KEEPER)
+    message(STATUS "ClickHouse keeper mode: ON")
```

Comment:
"Mode" sounds weird (here, but there are more places in this file). Printing "ClickHouse Keeper" is just fine.